### PR TITLE
Update kenya constituencies

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/demographics/kenya_constituency.json [hunk 1]
```diff
@@ -1995,13 +1995,13 @@
...
-      "DisplayText": "mbita",
+      "DisplayText": "suba north/mbita",
       "CodeType": "Normal",
       "NumericValue": 254,
-      "StringValue": "mbita",
+      "StringValue": "suba_north_mbita",
       "VisibleInCoda": true,
       "MatchValues": [
-        "mbita"
+        "suba north/mbita"
```
Comment (as2388) [inline code_schemes/demographics/kenya_constituency.json:2004]: Check which of these needs to match what it's called in CoreDataModules (e.g. stringvalues in mapping?)

**Review by as2388 — APPROVED** [review]

File: code_schemes/demographics/kenya_constituency.json (hunk 1 above)
Comment (DanielMwendwa) [reply code_schemes/demographics/kenya_constituency.json:2004]: string value it is: https://github.com/AfricasVoices/CoreDataModules/blob/3e2d90a37717f802c5b5169992305c4d94b68131/core_data_modules/analysis/mapping/participation_maps.py#L59

File: code_schemes/demographics/kenya_constituency.json [hunk 2]
```diff
@@ -1995,13 +1995,13 @@
     },
     {
       "CodeID": "code-c24cb56f",
-      "DisplayText": "mbita",
+      "DisplayText": "suba north/mbita",
       "CodeType": "Normal",
       "NumericValue": 254,
-      "StringValue": "mbita",
+      "StringValue": "suba_north_mbita",
```
Comment (as2388) [inline code_schemes/demographics/kenya_constituency.json:2001]:
```suggestion
      "StringValue": "suba_north",
```

File: code_schemes/demographics/kenya_constituency.json (hunk 1 above)
Comment (as2388) [inline code_schemes/demographics/kenya_constituency.json:2004]:
```suggestion
        "mbita",
        "suba north"
```

**Review by as2388 — APPROVED** [review]

**Pull request merged**